data/src/client.rs

```diff
@@ -644,8 +646,21 @@ impl Client {
 
                     if let Some(state) = self.chanmap.get_mut(channel) {
                         // Sends WHO to get away state on users.
-                        let _ = self.handle.try_send(command!("WHO", channel));
-                        state.last_who = Some(WhoStatus::Requested(Instant::now()));
+                        if self.isupport_parameters.get("WHOX").is_some() {
+                            let _ = self.handle.try_send(command!(
+                                "WHO",
+                                channel,
+                                "tcnf",
```

Comment:
I'm not sure we need `c` since we specify `WHO channel`

Reply:
I originally tried `"tnf"` because my reading of the `WHOX` documentation was that we would get the requested channel in the reply automatically, but when testing on Libera the individual `RPL_WHOSPCRPL` messages do not include the channel.  But, we could use `"tnf"` regardless if we use the token to distinguish the channel.

Reply:
Let's just leave it then, seems like servers aren't consistent on implementation, but adding it doesn't cause an additional channel arg to come across